src/GUI/efcpt/Services/PackageService.cs

```diff
@@ -0,0 +1,11 @@
+using ErikEJ.EfCorePowerTools.Options;
+
+namespace ErikEJ.EfCorePowerTools.Services;
+
+internal sealed class PackageService
+{
+    public PackageService(NuGetOptions options)
+    {
+        
```

Comment:
Is something missing here?

Reply:
oups!!! my last commit is not up to date

Reply:
> Is something missing here?

oups!!! my last commit is not up to date